### ISSUE TITLE
6 diferent amount of XCH in 6 computers all with same private key

### ISSUE THREAD

**Issue by FernandoCosta222** [Chia-Network/chia-blockchain#11019]
### What happened?

I have 6 computers (5 Harvester and 1 Wallet). 
Two XCH disappeared from my Wallet two weeks ago when I had 1.2.9 version.
Then apperad 1.3.1 (I didn´t instal 1.3.0 version) and I updated all my harvesters computers. After that all computers start to have diferent values on XCH Wallet. 
Than I creat on a sixth computer a Wallet.
But she also got a diferent amount of XCH (19:55 at 03/04/2022) on Wallet.

### Version

1.3.2

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
Then I updated for 1.3.2 version, but still the same problem:
- Harvester I 15,071469027396
- Harvester II 15,3675394696
- Harvester III 14,92056210094
- Harvester IV 15,490094764502
- Harvester V 16,795439935217
- Wallet 14,854038671712
There must be some bug because all have the same Private Key.
Can some one hellp me, please.
```

**Comment by paspy** [comment]
We have the same problem. It seems the new lightweight wallet can never figure out the correct balance compared to the previous full node wallet. Also, the syncing time for our private farming key takes an exceptionally long time to sync (even longer than the full node wallet starting from zero height)

**Comment by emlowe** [comment]
Please test the wallet by first syncing a local node and then syncing the wallet using that local node - this should give you the same value as versions prior to 1.3. Yes, we continue to work on issues with wallet syncing

**Comment by paspy** [comment]
> Please test the wallet by first syncing a local node and then syncing the wallet using that local node - this should give you the same value as versions prior to 1.3. Yes, we continue to work on issues with wallet syncing

I have tested this only using locally by deleting the wallet DB and syncing from the beginning (only using the local fully synced node) The balance is still incorrect plus takes hours likely more than the syncing time of the previous full node wallet. 

I try importing our farming key to a different PC (fully synced local node) to test the wallet synchronization. The transaction history is full of trash records with small amounts of mojos. probably from the dust storm. 

Screenshot below
<img width="1026" alt="image" src="https://user-images.githubusercontent.com/1343488/161894557-276dfc50-b2fa-442c-bc1e-dc3a78c4fd66.png">

**Comment by wallentx** [comment]
Yes, those appear to be transactions from dusting. How have you been able to determine the correct balance? After syncing to only a local full_node, how far off was your balance?

**Comment by paspy** [comment]
> 

I am assuming the full node wallet from version 1.2.11 gives me the correct balance since it's based on the original full node wallet. The balances differences are off every time the lightweight wallet is fully synced, from as many as 4 XCH to less than 1XCH. I don't see there is any pattern so far.

**Comment by FernandoCosta222** [comment]
I did what you said and now i have already all my computers with the same amount off XCH, but i feel that is missing 2 XCH. 
It is possible to select XCH transactions above certain amount? 

> Please test the wallet by first syncing a local node and then syncing the wallet using that local node - this should give you the same value as versions prior to 1.3. Yes, we continue to work on issues with wallet syncing

I did what you said and now i have already all my computers with the same amount off XCH, but i feel that is missing 2 XCH. 
It is possible to select XCH transactions above certain amount?

**Comment by FernandoCosta222** [comment]
> Please test the wallet by first syncing a local node and then syncing the wallet using that local node - this should give you the same value as versions prior to 1.3. Yes, we continue to work on issues with wallet syncing

I did what you said and now i have already all my computers with the same amount off XCH, but i feel that is missing 2 XCH. 
It is possible to select XCH transactions above certain amount?

**Comment by LeslieChiang** [comment]
something is winding up with the wallet and now I am also missing the xch payout from the pool.... zzz... how unreliable. Is my xch gone for good? Should I stop farming if not I really don't want to be feeding the effort into the air. Don't tell me all my plots is rendered useless now...

**Comment by punin** [comment]
I'm having the same issue with the latest client OSX M1 and balance is off by ~300 XCH when compared to explorers. But even xchscan.com and spacescan.io show different numbers (off by ~10 XCH). Deleted the wallet DB and now syncing, hopefully this helps..

**Comment by emlowe** [comment]
Possibly fixed in https://github.com/Chia-Network/chia-blockchain/pull/11338

**Comment by emlowe** [comment]
Please test with 1.3.5 as various fixes for this are in 1.3.5. Closing issue. Feel free to re-open if problems continue with 1.3.5
